frontend/src/components/pages/AdminDashboard/ViewDonationsAndCheckins.tsx

```diff
@@ -54,12 +73,21 @@ const ViewDonations = ({
         justifyContent="space-between"
         display={{ base: "inline", md: "flex" }}
       >
-        <Text
-          textStyle={isMobile ? "mobileHeader2" : "desktopHeader2"}
-          pt="2rem"
-        >
-          Scheduled donations
-        </Text>
+        <HStack justifyContent="space-between">
+          <Text
+            textStyle={isMobile ? "mobileHeader2" : "desktopHeader2"}
+            pt="2rem"
+          >
+            {isCheckInView ? "Fridge check-ins" : "Scheduled donations"}
+          </Text>
+
+          {isCheckInView && <CheckInAdminButtons />}
+        </HStack>
+        {isCheckInView ? (
+          <FridgeCheckInDescription />
+        ) : (
+          <FridgeFoodRescueDescription />
+        )}

```

Comment:
```suggestion
        {isCheckInView && (
          <FridgeCheckInDescription />
        )}
        {isAdminView && (
          <FridgeFoodRescueDescription />
        )}
```
If we have ternary operation, then we will see `fridgeFoodrescueDescription` in homepage as well which we do not want. 
<img width="748" alt="image" src="https://user-images.githubusercontent.com/37384882/160987354-280383a5-edee-4d89-9282-334583fd757b.png">

The suggested change should fix it